primefaces/src/main/java/org/primefaces/component/commandlink/CommandLinkRenderer.java

```diff
@@ -139,6 +137,10 @@ protected void encodeMarkup(FacesContext context, CommandLink link) throws IOExc
             renderPassThruAttributes(context, link, HTML.LINK_ATTRS);
         }
 
+        if (link.isDisabled()) {
+            writer.writeAttribute("tabindex", "-1", null);
+        }
+
```

Comment:
I moved this down because this needs to come AFTER `renderPassThroughAttributes` as the last `tabIndex` will win